setup.py

```diff
@@ -6,11 +6,11 @@
 
 install_requires = [
     "argparse",
-    "redis"
+    "redis <= 2.10.6",
```

Comment:
Would `redis < 3.0.0` be better, in the (unlikely) event that a bug-fix or security fix is released on the 2.X branch of the library?

Reply:
We are hitting an issue with the build server contextually bringing in redis 3.10.x which is making this restriction unusable.